frontend/app/controllers/Joiner.scala

```diff
@@ -44,7 +51,11 @@ trait Joiner extends Controller with ActivityTracking {
       request.path,
       Some(CopyConfig.copyDescriptionChooseTier)
     )
-    Ok(views.html.joiner.tierChooser(eventOpt, pageInfo))
+
+    val contentReferer = request.headers.get("referer")
```

Comment:
We probably should use the `Referer` constant here - I think it's already in scope.